### DISCUSSION TITLE
how to read many .vhdr files at a same time

### DISCUSSION THREAD

**Discussion by ghzhlchen** [bnicenboim/eeguana#225] [Q&A] [answered]
I have preprocessed my data using BVA and obtained n (where n is the participant number) .vhdr files. The example in the vignette demonstrates how to read one .vhdr file. Is it possible to read and process multiple .vhdr files simultaneously?

**Comment by bnicenboim** [comment, marked as answer]
Ah, that's quite straightforward, you read all the files, put them in a list and then `eeg_bind` them. See the short example in the help page of [eeg_bind](https://bruno.nicenboim.me/eeguana/reference/eeg_bind.html). I use the `purrr`  package there but it can be done with `lapply` or with just a for-loop.

(If you find the package useful, please cite it in your paper).
